modules/web/web-app/features/billing/plans/create/details/DetailsForm.tsx

```diff
@@ -53,6 +53,7 @@ export const DetailsForm: FC<Props> = ({ onCancel }) => {
       description: data.description,
       externalId: data.externalId,
       planType: PlanType[data.planType],
+      productFamilyExternalId: 'default',
```

Comment:
what should we do here instead of giving that static value?

Reply:
when creating a plan you have access to the selected product family in the url 
```
const { familyExternalId } = useTypedParams()
```

Reply:
ok thx I updated it, but I'm not sure to understand the purpose of that family mapping. Can you give an example please?

Reply:
A product family is a product line. It's useful when a company has different separated products that have their own plans that can be subscribed independently. 
For example you have https://www.hubspot.com/pricing/ , each product hub has a Professional Plan and an Enterprise Plan